Dockerfile.armhf

```diff
@@ -0,0 +1,20 @@
+FROM resin/armhf-alpine:3.5
+
+ENV GOPATH /go
+ENV PATH $PATH:$GOPATH/bin
+
+RUN apk update -y
+RUN apk add --no-cache git ca-certificates go g++
```

Comment:
I think a much smaller binary could be produced if you built and published the ARM binary through one Dockerfile and then used `ADD https://....` in another. Then you'd only need Alpine or Scratch as a base image and this could be tiny.